src/middleware.py

```diff
@@ -1,51 +1,22 @@
-from fastapi import Request, status
-from fastapi.responses import Response, JSONResponse
+import uuid
+from fastapi import Request
+from fastapi.responses import Response
 from starlette.middleware.base import BaseHTTPMiddleware
-import jwt
 
 
-SECRET_KEY = 'you will never guess'
-
-
-class sessionMiddleware(BaseHTTPMiddleware):
+class SessionMiddleware(BaseHTTPMiddleware):
     def __init__(self, app):
```

Comment:
If we don't have plans to expand `__init__` functionality of this class, we can omit this declaration.